portfolio/src/main/webapp/style.css

```diff
@@ -81,7 +100,7 @@ input[type=submit] {
 }
 
 /* Style form submit button when hovered over */
-input[type=submit]:hover {
+.link:hover, input[type=submit]:hover {
```

Comment:
newline for the new selector
https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation